### PR TITLE
chore: add task-2 webpack, linter, configs

### PR DESCRIPTION
**Pull request by pavelpiha**
task-2

### CODE REVIEW

File: src/App.tsx [hunk 1]
```diff
@@ -1,10 +1,10 @@
...
+import { SetStateAction, useState } from 'react';
+import './App.scss';
+import { Header } from './components/Header/Header';
 
-function App() {
-  const [searchEntry, setSearchEntry] = useState("");
-  const handleSubmit = (searchInput) => {
+function App(): any {
+  const [searchEntry, setSearchEntry] = useState('');
+  const handleSubmit = (searchInput: SetStateAction<string>): void => {
```
Comment (antonabramovich) [inline src/App.tsx]: Are you sure the type of the parameter is of type SetStateAction? Isn't it a `string`? Usually `SetStateAction` is not used explicitly, as it inherited from `useState` call.

File: src/components/SearchContainer/SearchContainer.tsx [hunk 2]
```diff
@@ -15,10 +15,7 @@ function SearchContainer({ handleIncrement, handleSubmit }: any) {
   }
 
   return (
-    <form
-      className="searchContainer"
-      onSubmit={updateSearch()}
-    >
+    <form className="searchContainer" onSubmit={updateSearch()}>
```
Comment (antonabramovich) [inline src/components/SearchContainer/SearchContainer.tsx]: I believe this function can be updated to `onSubmit={updateSearch}` and
```
function updateSearch(event) {
      event.preventDefault();
      handleIncrement();
      handleSubmit(searchEntry);
  }
```

**Review by antonabramovich — APPROVED** [review]

File: src/App.tsx (hunk 1 above)
Comment (pavelpiha) [reply src/App.tsx]: Yes, it should be string. Thank you

File: src/components/SearchContainer/SearchContainer.tsx (hunk 2 above)
Comment (pavelpiha) [reply src/components/SearchContainer/SearchContainer.tsx]: Thank you. Will fix it

**Pull request merged**